File: datasets/IR-Plag/original-kotlin-rest-java/case-05/plagiarized/L3/06/fungsi.java
package IR

import java.util.Scanner;

public class fungsi {

    public static void main(String[] args) {
         System.out.print("Enter an integer: ");
        
        Scanner sc = new Scanner(System.in);
        int   input;
        input =    sc.nextInt();//input integer
        terbalik(input);//memanggil fungsi reverse
    }

    public static void terbalik(int angka)  {
        while (angka != 0) {//looping sampai angka habis
             int   sisa ;
             sisa= angka % 10;//membagi bilangan tersebut dengan 0 sampai tidak ada hasil bagi
            System.out.print(sisa);
            angka = angka / 10;
        }
        
        System.out.println();
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L2/05/fungsi.kt
package IR

import java.util.Scanner

fun main() {
    println("Enter an integer: ")
    val sc = Scanner(System.`in`)
    val input = sc.nextInt() // input integer
    terbalik(input) // memanggil fungsi reverse
}

fun terbalik(angka: Int) {
    var temp = angka
    while (temp != 0) { // looping sampai angka habis
        val sisa = temp % 10 // membagi bilangan tersebut dengan 0 sampai tidak ada hasil bagi
        print(sisa)
        temp /= 10
    }
    println()
}

